templates/admin/status.php

```diff
@@ -0,0 +1,37 @@
+<?php
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+?>
+<div class="wrap hp-page">
+<h1 class="hp-page__title"><?php echo esc_html( hivepress()->get_name() ); ?> <span>/</span> <?php esc_html_e( 'Status', 'hivepress' ); ?></h1>
```

Comment:
Please correct the formatting, e.g. child HTML elements should have tab indents, also any output should be escaped (e.g. line 14, 19, 21, 23 - if WPCS works it should show warnings). Please go through the code one more time in detail before re-assigning this because I'd have to do this before merging anyway